.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
```

Comment:
I suggest adding `package-lock.json` here.

Reply:
Updated in 5207351ef035d3f82de502c709cdb899eaf0c478

Reply:
Just a note here - everybody will need to delete `node_modules` and `package-lock.json` on their local env and re-install the deps.